### PR TITLE
Added initial settings page and fixed infinite loading

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
Initial settings page allowing change of package directory (only visual at the moment, code needs to be placed to save the change). Fixed infinite loading error, this was due to page route change.

### CODE REVIEW

**Comment by GewoonJaap** [comment]
You gotta fix the two deepscan issues: https://deepscan.io/dashboard/#tid=10690&pid=13519&bid=230830&view=project&subview=pull-request&prid=458940

**Review by GewoonJaap — APPROVED** [review]

**Review by davwheat — CHANGES_REQUESTED** [review]
Fix the issues from DeepScan, please!
https://deepscan.io/dashboard/#tid=10690&pid=13519&bid=230830&view=project&subview=pull-request&prid=458940

File: src/pages/SplashScreen.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default function SplashScreen() {
   if (Config.get(ConfigKeys.state.setup_completed, false)) {
     console.log('Setup complete');
     setTimeout(() => {
-      Navigate('/manager');
+      Navigate('/home');
```
Comment (davwheat) [inline src/pages/SplashScreen.js:18]: We could change this string to use a value of `AllRoutes` which I recently exported from the [Routes.js](https://github.com/MSFS-Mega-Pack/MSFS2020-livery-manager/blob/217a26670bdb8a24e912458283cf48d4afd03f88/src/data/Routes.js#L7)

It'd prevent issues like this in the future.

**Review by davwheat — CHANGES_REQUESTED** [review]

File: src/pages/SplashScreen.js (hunk 1 above)
Comment (NinjaLabs-Dev) [reply src/pages/SplashScreen.js:18]: Changed 😄

**Review by davwheat — APPROVED** [review]
I'll merge this now. We can fix ESLint stuff in #18.

**Pull request merged**